pkg/webui/components/wizard/wizard.js

```diff
@@ -32,14 +34,19 @@ const reducer = (state, action) => {
         steps: action.steps,
       }
     case GO_TO_STEP:
+      const { snapshots: oldSnapshots, currentStep: oldStep } = state
```

Comment:
It would be good to use some inline commenting to explain the mechanics here. It's not easy to grasp on first glance.